jaeger-core/src/main/java/com/uber/jaeger/senders/UdpSender.java

```diff
@@ -32,6 +32,13 @@
   private Agent.Client agentClient;
   private ThriftUdpTransport udpTransport;
 
+  /**
+   * This constructor expects Jaeger running running on {@value #DEFAULT_AGENT_UDP_HOST} and port {@value #DEFAULT_AGENT_UDP_COMPACT_PORT}
+   */
+  public UdpSender() {
+    this(DEFAULT_AGENT_UDP_HOST, DEFAULT_AGENT_UDP_COMPACT_PORT, 0);
```

Comment:
is `maxPacketSize=0` valid?

Reply:
It is, look how it was called from the configuration class. I don't like that default values are defined in configuration classes.